skl/skl.go

```diff
@@ -132,16 +134,18 @@ func decodeValue(value uint64) (valOffset uint32, valSize uint32) {
 
 // NewSkiplist makes a new empty skiplist, with a given arena size.
 func NewSkiplist(arenaSize int64) *Skiplist {
-	buf, err := z.NewBufferWith(int(arenaSize), int(arenaSize), z.UseCalloc)
-	y.Check(err)
-	skl := NewSkiplistWithBuffer(buf, true)
-	return skl
+	buf := z.Calloc(int(arenaSize))
+	s := NewSkiplistWith(buf, true)
+	s.arena.Release = func() {
```

Comment:
How would someone set this `s.arena.Release()`? The `arena` is not exposed. You might want to move the `Release` method to the skiplist.

Reply:
Thanks. Fixed.